subsystems/provisioning/definitions.go

```diff
@@ -366,8 +370,8 @@ type Config struct {
 	// Computed from HotspotPrefix and Manufacturer
 	hotspotSSID string
 
-	// When true, it will disable power save for all wifi connections managed by NetworkManager.
-	DisableWifiPowerSave *bool `json:"disable_wifi_power_save"`
+	// If set, will explicitly enable or disable power save for all wifi connections managed by NetworkManager.
+	WifiPowerSave *bool `json:"wifi_power_save"`
```

Comment:
Double negatives are... weird. Changed to a postive. True == on == powersave. Let me know if that doesn't sound okay though.

Reply:
good call